CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.2.4.2
```

Comment:
Following the PVP, this should be 0.2.5 instead. Can you include a .cabal file update too?

Reply:
Sure